### ISSUE TITLE
clang-format configuration is required.

### ISSUE THREAD

**Issue by p1v0t** [RayTracing/InOneWeekend#78]
The potential contributors will be need clang-format configuration file to style code after editing. This is required for style consistency.

**Comment by hollasch** [comment]
I don't think we should add development tool requirements, particularly requiring all contributors to install some tool and learn how to use it on their particular platform. It's been a pretty simple matter to just follow the existing conventions, and break them where necessary (for example, wrapping at 86 columns for Markdown code blocks).
